### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Add CODEOWNERS

### CODE REVIEW

**Comment by emclaug2** [comment]
I think the CODEOWNERS file only takes effect when this file exists in the base of the PR.  So once this gets merged in, we can test it out with a dummy feature branch.

**Review by joebochill — APPROVED** [review]

**Pull request merged**
